src/Pages/home/Components/SearchTable.tsx

```diff
@@ -0,0 +1,82 @@
+import {
+  makeStyles,
+  Typography,
+  Toolbar,
+  TextField,
+  Button,
+} from "@material-ui/core";
+import React from "react";
+import { useState } from "react";
+
+const useStyles = makeStyles((theme) => ({
+  pageTitle: {
+    paddingLeft: theme.spacing(5),
+    padding: theme.spacing(3),
+    paddingBottom: theme.spacing(0),
+    "& .MuiTypography-subtitle2": {
+      color: "#979797",
+      marginTop: "12%",
+      fontSize: "16px",
+    },
+    "& .MuiTypography-h6": {
+      fontSize: "24px",
+    },
+  },
+  search: {
+    marginLeft: "20%",
+    marginTop: "2.5%",
+    "& .MuiTextField-root": {
+      margin: theme.spacing(1),
+      width: "200px",
+    },
+    fontSize: "20px",
+  },
+  buttonSearch: {
+    backgroundColor: "#EF5DA8",
+    color: "#ffffff",
+    textTransform: "none",
+    borderRadius: "20px",
+    fontSize: "18px",
+  },
+}));
+
+const SearchTables = (props: any) => {
```

Comment:
I don't suggest to defined every type that any type (it has no meaning when others debug). Please define type if it can.